angular-ngrx-scss/src/app/state/user/user.effects.spec.ts

```diff
@@ -2,18 +2,19 @@ import { TestBed } from '@angular/core/testing';
 import { provideMockActions } from '@ngrx/effects/testing';
 import { Action } from '@ngrx/store';
 import { Observable, of } from 'rxjs';
+import { fetchUserData, fetchUserDataSuccess } from '.';
 import { UserService } from '../../user/services/user.service';
-import { fetchUserData, fetchUserDataSuccess } from './user.actions';
 import { UserEffects } from './user.effects';
 import { UserState } from './user.state';
 
 describe('UserEffects', () => {
   let actions$: Observable<Action>;
   let effects: UserEffects;
   let userServiceMock: any;
+
   beforeEach(() => {
     userServiceMock = jasmine.createSpyObj('UserService', {
-      getUserInfo: () => {
+      getAuthenticatedUserInfo: () => {
```

Comment:
This and line 56 can potentially be changed back if we don't keep both calls.

Reply:
We should discuss this later. I think I have some missing context.